src/tag.test.miscellaneous.ts

```diff
@@ -0,0 +1,59 @@
+import { html } from 'lit';
+import { expect, test } from './playwright/test.js';
+
+test('registers itself', { tag: '@miscellaneous' }, async ({ mount, page }) => {
+  await mount(() => html`<glide-core-tag label="Label"></glide-core-tag>`);
+
+  const host = page.locator('glide-core-tag');
+
+  await expect(host).toBeRegistered('glide-core-tag');
+});
+
+test(
+  'can be focused programmatically',
+  { tag: '@miscellaneous' },
+  async ({ callMethod, mount, page }) => {
+    await mount(
+      () => html`<glide-core-tag label="Label" removable></glide-core-tag>`,
+    );
+
+    const host = page.locator('glide-core-tag');
+
+    await callMethod(host, 'focus');
+
+    await expect(page.getByRole('button')).toBeFocused();
```

Comment:
Doesn't make  a difference. But other functionality tests do selection up front then pass the locator to `expect()`.